bitmapcontainer.go

```diff
@@ -31,8 +33,8 @@ func newBitmapContainerwithRange(firstOfRun, lastOfRun int) *bitmapContainer {
 	if bc.cardinality == maxCapacity {
 		fill(bc.bitmap, uint64(0xffffffffffffffff))
 	} else {
-		firstWord := firstOfRun / 64
-		lastWord := lastOfRun / 64
+		firstWord := firstOfRun >> log2WordSize
+		lastWord := lastOfRun >> log2WordSize
```

Comment:
Here this will result in different compiled code because firstOfRun and lastOfRun are signed integers.